### PR TITLE
CCE and CIS codes for SLE 12/15 rules which belong to CIS group 5.2 c…

### PR DESCRIPTION
**Pull request by rumch-se**
…onfigure ssh server

#### Description:

- _Assignment of CCE/CIS numbers or CIS numbers update_

#### Rationale:

- Some of SLE 12/15 rules did not have CCE/CIS numbers assigned or some of CIS numbers were not correct 

#### Review Hints:

- _Review hints here. Replace this text. Don't use the italics format!_

- _Use this optional section to give any relevant information which could help the reviewer to more quickly and assertively understand and test the changes._

- _Good examples are useful commands, if it is better to review all commits together or in a suggested sequence, any relevant discussion in other PRs or issues, etc._

### CODE REVIEW

**Comment by rumch-se** [comment]
Hello @marcusburghardt
I have done rebase.
Have a nice day
Rumen

**Comment by rumch-se** [comment]
Hello @marcusburghardt
I have done rebase.
Have a nice day
Rumen

File: shared/references/cce-sle12-avail.txt [hunk 1]
```diff
@@ -1,31 +1,3 @@
...
-CCE-92272-4
-CCE-92273-2
-CCE-92274-0
-CCE-92275-7
-CCE-92276-5
-CCE-92277-3
-CCE-92278-1
-CCE-92279-9
-CCE-92280-7
-CCE-92281-5
```
Comment (marcusburghardt) [inline shared/references/cce-sle12-avail.txt:28]: You are probably removing more CCEs than necessary in this PR. Could you check, please?

File: shared/references/cce-sle15-avail.txt [hunk 2]
```diff
@@ -1,27 +1,3 @@
...
-CCE-91388-9
-CCE-91389-7
-CCE-91390-5
-CCE-91391-3
-CCE-91392-1
-CCE-91393-9
-CCE-91394-7
-CCE-91395-4
-CCE-91396-2
-CCE-91397-0
```
Comment (marcusburghardt) [inline shared/references/cce-sle15-avail.txt:24]: You are probably removing more CCEs than necessary in this PR. Could you check, please?

**Review by marcusburghardt — CHANGES_REQUESTED** [review]

**Comment by rumch-se** [comment]
Good morning @marcusburghardt 
I have done necessary changes and now CCE codes taken from the files with available numbers correspond to the codes used in the rules.
Have a nice day
Rumen

**Review by marcusburghardt — APPROVED** [review]

**Pull request merged**